### PR TITLE
[foss/2015a] Python 2.7.9 + deps (OK)

### PR DESCRIPTION
**Pull request by gjbex**
Easyconfigs (foss 2015a) for:
* zlib
* bzip2
* Szip
* ncurses
* libreadline
* Python-bare
* Python

### CODE REVIEW

**Comment by gjbex** [comment]
Test report by @gjbex
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
Linux red hat enterprise linux computenode 6.4, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.6
See https://gist.github.com/ad6ff4ebcbe5f0b4aa20 for a full test report.

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by gjbex** [comment]
Test report by @gjbex
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
Linux red hat enterprise linux computenode 6.4, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.6
See https://gist.github.com/00226d60a1417fae73d8 for a full test report.

**Comment by gjbex** [comment]
Test report by @gjbex
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
Linux red hat enterprise linux computenode 6.4, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.6
See https://gist.github.com/e4bf10646e0e42e8fd31 for a full test report.

File: easybuild/easyconfigs/p/Python/Python-2.7.9-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
+#   ('OpenSSL', '1.0.1k'),  # OS dependency should be preferred if the os version is more recent then this version, it's
+#   nice to have an up to date openssl for security reasons
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev')]
+
+# order is important!
+# Version updated 08/JULY/2014
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.9-foss-2015a.eb]: please fix this comment to mention `Jan 13th 2015` (cfr. #1298), and please do the same in the `intel/2015a` easyconfig?

**Comment by boegel** [comment]
@gjbex: one very minor remark

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7bb30ffbb4c4b0ff76b1 for a full test report.

**Comment by boegel** [comment]
@gjbex: ignore the failed tests, for some reason Python packages aren't happy when they're being installed from a directory that has `[`, `]` characters in its name (i.e., `$TMPDIR` in a PBS job)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5fac4828657296946dc6 for a full test report.

**Comment by boegel** [comment]
retesting this after #1316 by @rjeschmi was merged in, which affects `foss/2015a`, just as sanity check that things didn't break...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5c4a5545b6b37dd72a5d for a full test report.

**Comment by boegel** [comment]
still looking good, so no reason to hold this back

Thanks @gjbex!

**Pull request merged**